services/network/public/cpp/content_security_policy/content_security_policy.cc

```diff
@@ -144,6 +149,11 @@ CSPDirectiveName ToCSPDirectiveName(base::StringPiece name) {
   if (base::EqualsCaseInsensitiveASCII(name, "navigate-to")) {
     return CSPDirectiveName::NavigateTo;
   }
+#if BUILDFLAG(IS_COBALT)
+  if (base::EqualsCaseInsensitiveASCII(name, "h5vcc-location-src")) {
```

Comment:
As above, leave this in place, but can we add `cobalt-location-src` as an alternative name, so we can start migrating to more compliant name at least ? Once client supports both naming variants, we can add it in the server.